frontend/src/pages/privacy.tsx

```diff
@@ -31,12 +31,18 @@ const ToS: FC = () => {
           </header>
           <main>
             <h1>Privacy Policy</h1>
+
             <p>
               April 1, 2020. There is no prior version of this privacy policy.
             </p>
 
             <h2>Introduction</h2>
 
+            <p>
+              We invite you to <a href="/previewpolicies">preview</a> our
+              updated Terms of Use, which will go into effect on April 1, 2021.
```

Comment:
Looks like a cut-n-paste error - this is the privacy policy, not terms of use:

> We invite you to preview [link to cellxgene.cziscience.com/previewpolicies] our updated **Privacy Policy,** which will go into effect on April 1, 2021.

Reply:
Oh!! Yes yes 🙏 Thank you for the eagle eye, @brianraymor ! Will update now! 🦅

Reply:
Updated! Thanks again!

<img width="686" alt="Screen Shot 2021-03-15 at 1 15 14 PM" src="https://user-images.githubusercontent.com/6309723/111215703-abdb6880-8590-11eb-9492-ad52040723b5.png">